src/Ignitor/SignalsListener/index.ts

```diff
@@ -36,16 +50,13 @@ export class SignalsListener {
    */
   public listen(callback: () => Promise<void>) {
     this.onCloseCallback = callback
-    if (process.env.pm_id) {
-      process.once('SIGINT', this.kill)
-    }
-
-    process.once('SIGTERM', this.kill)
+    process.on('SIGINT', this.kill)
+    process.on('SIGTERM', this.kill)
 
     /**
      * Cleanup on uncaught exceptions.
      */
-    process.once('uncaughtException', (error) => {
+    process.on('uncaughtException', (error) => {
```

Comment:
that's not directly related to this PR, so I can do it in a separate one if you prefer. I changed to `process.on` so that in the REPL, all uncaught exceptions are handled (not only the first one)